SConstruct

```diff
@@ -236,6 +237,13 @@ opts.Add(BoolVariable("engine_update_check", "Enable engine update checks in the
 opts.Add(BoolVariable("steamapi", "Enable minimal SteamAPI integration for usage time tracking (editor only)", False))
 opts.Add("cache_path", "Path to a directory where SCons cache files will be stored. No value disables the cache.", "")
 opts.Add("cache_limit", "Max size (in GiB) for the SCons cache. 0 means no limit.", "0")
+opts.Add(
+    BoolVariable(
+        "redirect_build_objects",
+        "Enable redirecting built objects/libraries to `bin/obj/` to declutter the repository.",
+        True,
+    )
+)
```

Comment:
It would be possible to make it a `build_objects_dir` (or similar), with a default value of `bin/obj`. Passing the empty string would place the objects in the source directory.
I don't _expect_ anybody to want a different build dir, but you never know :D

Reply:
Ohh, good call!

Reply:
Actually, gonna walk back on this for now; it's a bit more involved to integrate than I expected, so I'll leave them hardcoded for the time being